doc/man3/RAND_DRBG_set_callbacks.pod

```diff
@@ -82,12 +82,14 @@ free the buffer allocated previously by get_entropy().
 The values B<out> and B<outlen> are the random buffer's  address and length,
 as returned by the get_entropy() callback.
 
-The B<get_nonce>() and B<cleanup_nonce>() callbacks are analogous to
-B<get_entropy>() and B<cleanup_entropy>() and are used by the B<drbg>
-to request random bits for the nonce.
-These two callbacks are optional, it is allowed to pass NULL function pointers.
-It is also allowed to reuse the B<get_entropy>() and B<cleanup_entropy>()
-callbacks as B<get_nonce>() and B<cleanup_nonce>() callbacks.
+The B<get_nonce>() and B<cleanup_nonce>() callbacks are used in the case
+where the DRBG uses a derivation function for reseeding.
+The callbacks are analogous to get_entropy() and cleanup_entropy(),
+except for the missing prediction_resistance flag.
+
+If the derivation function is disabled, then no nonce is used for reseeding,
```

Comment:
As far as I know, get_nonce is only used for instantiation, never for reseeding?

Reply:
That's possible. I'll have to look it up again...

Reply:
Corrected: 9f0207f2887b08b1b5c888b8471107280beb48d4